src/compiler/checker.ts

```diff
@@ -18678,6 +18678,7 @@ namespace ts {
             if (node.expression.kind === SyntaxKind.SuperKeyword) {
                 const superType = checkSuperExpression(node.expression);
                 if (isTypeAny(superType)) {
+                    forEach(node.arguments, checkExpression); // Still visit arguments so they get marked for visibility, etc

```

Comment:
@weswigham this should be `forEach(node.arguments, n=>{checkExpression(n);});` 

the return value from `checkExpression` is short-circuiting after the first argument.

Reply:
Repro in https://github.com/Microsoft/TypeScript/issues/24729#issuecomment-395808993

Reply:
D'oh.